base_x: .7
base_y: .45
laser_x: .37
